### PR TITLE
Chore/navbar wrapper

### PR DESCRIPTION
**Pull request by rcdjasmin**
- Wrapped pages in a NavBarWrapper to render navbar
- Switched <Link> to <NavLink>
- Added active state using activeClassName
- Logout button pushed to the bottom

### CODE REVIEW

File: src/helpers/NavBarWrapper.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import NavBar from '../components/NavBar'
+
+function NavBarWrapper(Component) {
```
Comment (Weiyuan-Lane) [inline src/helpers/NavBarWrapper.js:4]: @rcdjasmin Looking at the other contents now, I get a better idea now. A suggestion to make this better

```javascript
function NavBarWrapper(Component) {
    return function() {
        return (
            <React.Fragment>
                <NavBar/>
                <main>
                  <Component/>
                </main>
            </React.Fragment>
        )
    }
}
```

We can then remove `main` element repeated on the other page components

File: src/components/DonorOverview.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    <button onClick={()=>setEditing(true)} className="btn btn-outline-primary">Edit donor</button>
+  )
+
+  /*render*/
+  return(
+      <main>
+      <h1>Donor Records</h1>
+      <div className="breadcrumbs">
+        <p>Donor List  >  <strong>{donor.name}</strong></p>
+        { editing ? <div className="btn-toolbar mr-2"><SaveButton/><CancelButton/></div> : <EditButton/> }
```
Comment (Weiyuan-Lane) [inline src/components/DonorOverview.js]: For this one, the save button and cancel will need some spacing here. let's add a small margin as a spacing between the buttons

**Review by Weiyuan-Lane — CHANGES_REQUESTED** [review]

File: src/components/DonorOverview.js [hunk 3]
```diff
@@ -34,7 +36,7 @@ const DonorRecords = () => {
   )
 
   const CancelButton = () => (
-    <button onClick={()=>setEditing(false)} className="btn btn-outline-primary">Cancel</button>
+    <button onClick={()=>setEditing(false)} className="btn btn-link">Cancel</button>
```
Comment (rcdjasmin) [inline src/components/DonorOverview.js]: @Weiyuan-Lane Changed the Cancel button to btn-link to make the difference between 2 buttons clearer

Comment (rcdjasmin) [inline src/components/DonorOverview.js:38]: @Weiyuan-Lane changed the Cancel button to btn-link to make the difference between the 2 buttons clearer. I accidentally included this in the navbar-related commit

**Review by Weiyuan-Lane — APPROVED** [review]
[LGTM](https://lgtm.lol/i/493)

**Pull request merged**